include/aspect/parameters.h

```diff
@@ -68,6 +68,7 @@ namespace aspect
         iterated_Advection_and_Stokes,
         single_Advection_iterated_Stokes,
         no_Advection_iterated_Stokes,
+        single_Advection_DC_iterated_Stokes,
```

Comment:
I would prefer the following name:
```suggestion
        single_Advection_iterated_defect_correction_Stokes,
```
It is easier to understand and more consistent with the other schemes.